### DISCUSSION TITLE
[nodriver] cookies behaving weirdly

### DISCUSSION THREAD

**Discussion by splake975** [ultrafunkamsterdam/undetected-chromedriver#1957] [Q&A] [answered]
I have heard talks about nodriver cookies not being able to be saved and i dont see anyone saying they can. is this true?

I was using cookies.load(file=something) and cookies.save() two days ago, but now my program is broken and I want to find a way to more permanently get this working. 

my file in the argument of load and save was "bla blah blah/Profile 1/Network/Cookies". do I want link the Cookies file? this file doesnt have any extension, I dont know what it does, idk where I can read what it does. 

the documentation [here](https://ultrafunkamsterdam.github.io/nodriver/nodriver/classes/browser.html) says that if i leave the arguments of save() and load() blank it saves it to "session.dat", I have no idea what that is and if it even works or not. 

I try to use cookies.save(file=something), but my code sometimes gets stuck on this line, and sometimes doesnt get stuck. I have no idea why.  

also. if cookies arent being loaded or stored properly, how is it that on certain sites I can log in, close chrome, start a new chrome session and I remain logged in?

**Comment by ilkecan** [comment, marked as answer]
Try using a more recent version of the browser. I was having the same problem with Chromium v123-124 and upgrading to v127 fixed the problem for me.
